librocksdb_sys/src/generate.py

```diff
@@ -34,7 +34,7 @@
             if task[2].match(line):
                 begin = True
                 print('#[derive(Copy, Clone, Debug, Eq, PartialEq)]')
-                print('#[repr(C)]')
+                print('#[repr(u32)]')
```

Comment:
why change from "repr(C)" to "repr(u32)"?

Reply:
The enums are defined as `uint32_t` in the C++ code